examples/chat/main.go

```diff
@@ -57,7 +54,7 @@ func main() {
 
 	net, err := builder.Build()
 	if err != nil {
-		glog.Fatal(err)
+		log.Fatal().Err(err)
```

Comment:
for warning/error/fatal, we should add file and line number to log to help debug:
```
log.Logger = log.With().Caller().Logger()
log.Info().Msg("hello world")

// Output: {"level": "info", "message": "hello world", "caller": "/go/src/your_project/some_file:21"}
```
https://github.com/rs/zerolog#add-file-and-line-number-to-log

Reply:
it doesn't seem to work that well:
```
{"level":"info","time":"2018-09-19T10:29:59-04:00","caller":"/Users/jack/go/pkg/mod/github.com/rs/zerolog@v1.8.0/event.go:116","message":"Setting up NAT traversal for address: tcp://127.0.0.1:49197"}
{"level":"warn","error":"no NAT found","time":"2018-09-19T10:30:09-04:00","caller":"/Users/jack/go/repos/noise/network/nat/plugin.go:43","message":"Unable to discover gateway"}
{"level":"info","time":"2018-09-19T10:30:09-04:00","caller":"/Users/jack/go/pkg/mod/github.com/rs/zerolog@v1.8.0/event.go:116","message":"Listening for peers on tcp://127.0.0.1:49197."}
{"level":"info","time":"2018-09-19T10:30:09-04:00","caller":"/Users/jack/go/pkg/mod/github.com/rs/zerolog@v1.8.0/event.go:116","message":"Setting up NAT traversal for address: tcp://127.0.0.1:49375"}
{"level":"warn","error":"no NAT found","time":"2018-09-19T10:30:19-04:00","caller":"/Users/jack/go/repos/noise/network/nat/plugin.go:43","message":"Unable to discover gateway"}
{"level":"info","time":"2018-09-19T10:30:19-04:00","caller":"/Users/jack/go/pkg/mod/github.com/rs/zerolog@v1.8.0/event.go:116","message":"Listening for peers on tcp://127.0.0.1:49375."}
```

Reply:
related to this issue that was fixed july https://github.com/rs/zerolog/pull/89

v1.8 that we're using is from may, but they just released v1.9.0 7 hours ago.. updating the version fixes the line number issue